wiki/FitNesseRoot/HsacExamples/SlimTests/HttpTests.wiki

```diff
@@ -101,17 +101,32 @@ The commands/keywords (i.e. public methods) added by !-nl.hsac.fitnesse.fixture.
+|register prefix <prefix> for namespace <url>                                              |Defines a prefix to be used to identify a namespace in subsequent 'xPath' evaluation.                                                                                                                |
+|xPath int <expression>                                                                    |Evaluates the expression against the last response received, returning a whole number (allowing [[relative value tests][.FitNesse.UserGuide.WritingAcceptanceTests.SliM.ValueComparisons]]).         |
+|xPath double <expression>                                                                 |Evaluates the expression against the last response received, returning a floating point number (allowing [[relative value tests][.FitNesse.UserGuide.WritingAcceptanceTests.SliM.ValueComparisons]]).|
+|raw xPath <expression>                                                                    |Evaluates the expression against the last response received, ignoring any (SOAP) faults received (which cause 'xPath <expression>' to throw an exception).                                           |
+|create file <fileName> from base64 content of <expression>                                |Evaluates the expression against the last response received, expecting the found element to contain a base64 encoded value. That value will be saved to a file and a link to the file returned.      |
+|repeat until xPath <expression> is <value>                                                |If 'expression' does not evaluates to 'value' for the last response: repeats last request until it does. Do not forget to set 'repeat at most times', or else the loop may run endlessly!            |
+|validate response against xsd <XSD Schema>                                                |Validates the response xml against the given XSD Schema                                                                                                                                              |
+|validate response against xsd file <XSD File>                                             |Validates the response xml against the XSD Schema in the given XSD File                                                                                                                              |
+|'''WS Security configuration options'''                                                                                                                                                                                                                                                         |
+|ws sec apply soap signature before sending <true/false>                                   |Enable/disable appending a soap signature to the ws-sec header. This needs                                                                                                                           |
```

Comment:
My previous comment  about the incomplete sentence was at the wrong line. I was referring to 
> ...header. This needs